code/smart-mobility/smart-mobility.js

```diff
@@ -177,3 +177,129 @@ $(document).on("knack-scene-render.scene_281", function () {
 $(document).on("knack-scene-render.scene_284", function () {
   disableBreadCrumbsNonAdmin();
 });
+
+  //***MANAGE WALLET AND MAKE PAYMENT BUTTONS***
+$(document).on('knack-view-render.view_804', function(event, page) {
+  // create large button on go back to Portal page
+  bigButton('manage-wallet', 'view_804', "https://austin-tx-austin-tx.uat.cityba.se/welcome", "credit-card", "Manage Wallet");
+  // bigButton('manage-wallet', 'view_804', "https://atd.knack.com/smart-mobility#living-streets-applicant-portal/", "credit-card", "Manage Wallet");
+});
+$(document).on('knack-view-render.view_806', function(event, page) {
+  // create large button on go back to Portal page
+  // *** see the getCitybaseButton function for the invocation of the make payment bigButton function
+  //   bigButton('make-payment', 'view_806', "https://atd.knack.com/smart-mobility#living-streets-applicant-portal/", "arrow-right", "Make Payment");
+});
+
+/** 
+ * Citybase Integration
+ */
+
+function getCitybaseButton(payload, viewId) {
+  fetch(
+    "https://invoice-service-austin-tx.uat.cityba.se/invoices/austin_tx_transportation/street_banner",
+    {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify(payload),
+    },
+  )
+    .then((response) => {
+      console.log(response.url);
+      bigButton(
+        "make-payment",
+        viewId,
+        response.url,
+        "arrow-right",
+        "Make Payment",
+      );
+    })
+    .catch((error) => {
+      console.error("Error:", error);
+    });
+}
+
+var knackUserToken = Knack.getUserToken();
+
+var headers = {
+  "X-Knack-Application-Id": "618ad3322d11b4002169a6f9",
```

Comment:
I get a little spooked seeing a magic number like this coded right into app like this, but I also recognize the constraints of how we have to deploy this JS and crafting some mechanism to get environment variables or similar working is not possible or worth it. 

If that is the case, and it is best to leave it hard-coded in the JS, what would you think about defining this in a capitalized variable name up top? `KNACK_APPLICATION_ID` or similar?